src/rgw/rgw_op.cc

```diff
@@ -2214,6 +2215,13 @@ void RGWGetObj::execute(optional_yield y)
   std::unique_ptr<RGWGetObj_Filter> run_lua;
   map<string, bufferlist>::iterator attr_iter;
 
+  bool rgw_perf_counters_cache = s->cct->_conf.get_val<bool>("rgw_perf_counters_cache");
+  std::string labels = ceph::perf_counters::key_create("rgw_get", {{"Bucket", s->bucket_name}, {"User", s->user->get_display_name()}});
```

Comment:
the user label should probably use the user's id; i don't think the display name is required to be unique (i.e. several users could have the same name)